### DISCUSSION TITLE
cloudflare zerotrust下無法下載本地/內部SMB掛載檔案

### DISCUSSION THREAD

**Discussion by amath1986** [AlistGo/alist#7441] [Invalid question]
在利用cloudflare的zerotrust掛載alist，在下載/預覽自己內部NAS或本地掛載的檔案時，會變成{"code":401,"message":"expire missing","data":null}，若是下載百度或dropbox的掛載網盤是無問題。
alist version:3.37.4

<img width="1239" alt="image" src="https://github.com/user-attachments/assets/e113c3dc-721c-44a3-9792-9a03071a8cd5">

**Comment by anwen-anyi** [comment]
可能是在维护有关？
- https://www.cloudflarestatus.com

**Reply by amath1986** [reply]
感覺不像，因為直掛CDN出去好像正常，並且我發現應該所有PREVIEW都不行

**Comment by amath1986** [comment]
Solution: 1. Update cloudflared process to the latest version.
2. Caching Level select to "No query string"
